src/db/migrations/20190325172457-create-users.js

```diff
@@ -29,7 +29,7 @@ export default {
       },
       password: {
         type: Sequelize.STRING,
-        allowNull: false,
+        allowNull: true,
```

Comment:
Please, I need clarity on why you changed `allowNull`  to `TRUE` and does it mean that a user can be created without providing a password.

Reply:
Exactly. When a user registers with a third-party application like Facebook, his password would not be provided by Facebook. However, he is able to create/edit password on his profile.

Reply:
Oh, I get it now. Nice one